charts/incubator/alist/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: xhofe/alist
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
Please open a PR on truecharts/containers, and add this image in mirror folder 
Then update this image to the tccr.io/truecharts repo